src/app/page.tsx

```
@@ -2,29 +2,35 @@ import { ConversationModel } from "@/lib/models/conversationModel";
-      </ul>
+      <h1>{websiteTitle}</h1>
+
+      {/* Card Wrapper for Conversations */}
+      <div className="conversation-card">
+        <ul className="conversation-list">
+          {conversations.map((conversation) => (
+            <li key={conversation.id} className="conversation-item">
+              <details>
+                <summary>{conversation.question}</summary>
+                <p className="answer-text">
+                  <span dangerouslySetInnerHTML={{ __html: conversation.answer }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)